packages/luvit/build.sh

```diff
@@ -0,0 +1,25 @@
+TERMUX_PKG_HOMEPAGE=https://luvit.io
+TERMUX_PKG_DESCRIPTION="Asynchronous I/O for Lua"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=2.18.1
+TERMUX_PKG_SRCURL=https://github.com/luvit/luvit/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=b792781d77028edb7e5761e96618c96162bd68747b8fced9a6fc52f123837c2c
+TERMUX_PKG_BUILD_DEPENDS="luvi"
+TERMUX_PKG_SUGGESTS="luvi, lit"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_NO_STRIP=true
+
+termux_step_configure() {
+	curl -Lo- https://github.com/luvit/lit/raw/master/get-lit.sh | sh
```

Comment:
Here I understand why we're pulling the `get-lit.sh` from the external repo.
That makes sense, this is `luvit`'s build.
But it should still be pinned to the `lit` version we are shipping.
```suggestion
	curl -Lo- https://github.com/luvit/lit/raw/$(. $TERMUX_SCRIPTDIR/packages/lit/build.sh; echo ${TERMUX_PKG_VERSION#*:})/get-lit.sh | sh
```
(See MPV for reference)
https://github.com/termux/termux-packages/blob/d2cb5482fa8c74b2f4ff27286470991d2289c9b5/packages/mpv/build.sh#L37

Reply:
It won't make any difference, unless exact Lit version is required to build Luvit. And it's only used to build Luvit.

Reply:
Actually I changed my mind, let me do that real quick.

Reply:
I still think we should pin it to the same version as our `lit` package.